charts/authentik/values.yaml

```diff
@@ -118,175 +195,740 @@ authentik:
+    behavior: {}
+      # scaleDown:
+      #   stabilizationWindowSeconds: 300
+      #   policies:
+      #     - type: Pods
+      #       value: 1
+      #       periodSeconds: 180
+      # scaleUp:
+      #   stabilizationWindowSeconds: 300
+      #   policies:
+      #     - type: Pods
+      #       value: 2
+      #       periodSeconds: 60
+    # -- Configures custom HPA metrics for the authentik worker
+    # Ref: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+    metrics: []
+
+  ## authentik worker Pod Disruption Budget
+  ## Ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/
+  pdb:
+    # -- Deploy a [PodDistrubtionBudget] for the authentik worker
+    enabled: false
+    # -- Labels to be added to the authentik worker pdb
+    labels: {}
+    # -- Annotations to be added to the authentik worker pdb
     annotations: {}
     # -- Number of pods that are available after eviction as number or percentage (eg.: 50%)
     # @default -- `""` (defaults to 0 if not specified)
     minAvailable: ""
     # -- Number of pods that are unavailable after eviction as number or percentage (eg.: 50%)
-    ## Has higher precedence over `pdb.worker.minAvailable`
+    ## Has higher precedence over `worker.pdb.minAvailable`
     maxUnavailable: ""
 
-livenessProbe:
-  # -- enables or disables the livenessProbe
-  enabled: true
-  httpGet:
-    # -- liveness probe url path
-    path: /-/health/live/
-    port: http
-  initialDelaySeconds: 5
-  periodSeconds: 10
-
-startupProbe:
-  # -- enables or disables the livenessProbe
-  enabled: true
-  httpGet:
-    # -- liveness probe url path
-    path: /-/health/live/
-    port: http
-  failureThreshold: 60
-  periodSeconds: 5
-
-readinessProbe:
-  enabled: true
-  httpGet:
-    path: /-/health/ready/
-    port: http
-  periodSeconds: 10
+  ## authentik worker image
+  image:
+    # -- Repository to use to the authentik worker
+    # @default -- `""` (defaults to global.image.repository)
+    repository: "" # defaults to global.image.repository
+    # -- Tag to use to the authentik worker
+    # @default -- `""` (defaults to global.image.tag)
+    tag: "" # defaults to global.image.tag
+    # -- Digest to use to the authentik worker
+    # @default -- `""` (defaults to global.image.digest)
+    digest: "" # defaults to global.image.digest
+    # -- Image pull policy to use to the authentik worker
+    # @default -- `""` (defaults to global.image.pullPolicy)
+    pullPolicy: "" # defaults to global.image.pullPolicy
+
+  # -- Secrets with credentials to pull images from a private registry
+  # @default -- `[]` (defaults to global.imagePullSecrets)
+  imagePullSecrets: []
+
+  # -- Environment variables to pass to the authentik worker. Does not apply to GeoIP
+  # See configuration options at https://goauthentik.io/docs/installation/configuration/
+  # @default -- `[]` (See [values.yaml])
+  env: []
```

Comment:
Same as above, not sure if we want to allow this, while a lot of config options are only needed for either component, it also allows a lot of room for misconfiguration

Reply:
For this particular option, I'm thinking that most people use the `authentik` key anyway. But as with the image, we also provide `global.env`.